pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -1026,6 +1035,90 @@ func registriesConfigIgnition(templateDir string, controllerConfig *mcfgv1.Contr
 	return &registriesIgn, nil
 }
 
+func (ctrl *Controller) createDefaultContainerRuntimeMC() error {
+	var configMapName = "crio-default-container-runtime"
+
+	// Check if the crio-default-container-runtime config map exists in the openshift-machine-config-operator namespace
+	defaultContainerRuntimeCM, err := ctrl.kubeClient.CoreV1().ConfigMaps(ctrl.namespace).Get(context.TODO(), configMapName, metav1.GetOptions{})
+	defaultRuntimeCMIsNotFound := errors.IsNotFound(err)
```

Comment:
this variable `defaultRuntimeCMIsNotFound` can be deleted and errors.IsNotFound used on line 1044.